rest/src/main/kotlin/systems/ajax/englishstudytelegrambot/grpc/GrpcWordService.kt

```diff
@@ -0,0 +1,96 @@
+package systems.ajax.englishstudytelegrambot.grpc
+
+import net.devh.boot.grpc.server.service.GrpcService
+import org.slf4j.LoggerFactory
+import reactor.core.publisher.Mono
+import reactor.kotlin.core.publisher.toMono
+import systems.ajax.englishstudytelegrambot.mapper.toWordResponse
+import systems.ajax.englishstudytelegrambot.dto.request.CreateWordDtoRequest as ServiceCreateWordDtoRequest
+import systems.ajax.englishstudytelegrambot.service.WordService
+import systems.ajax.service.CreateWordDtoRequest
+import systems.ajax.service.DeleteWordRequest
+import systems.ajax.service.DeleteWordResponse
+import systems.ajax.service.GetFullInfoAboutWordRequest
+import systems.ajax.service.GetFullInfoAboutWordResponse
+import systems.ajax.service.ReactorWordServiceGrpc
+import systems.ajax.service.SaveNewWordRequest
+import systems.ajax.service.SaveNewWordResponse
+import systems.ajax.service.UpdateWordTranslateRequest
+import systems.ajax.service.UpdateWordTranslateResponse
+
+@GrpcService
+class GrpcWordService(
+    val wordService: WordService
+) : ReactorWordServiceGrpc.WordServiceImplBase() {
+
+    override fun saveNewWord(request: Mono<SaveNewWordRequest>): Mono<SaveNewWordResponse> =
+        request.flatMap {
+            wordService.saveNewWord(it.libraryName, it.telegramUserId, it.createWordDtoRequest.toServiceDto())
+        }.map {
+            SaveNewWordResponse.newBuilder().apply {
+                successBuilder.setWord(it.toWordResponse())
+            }.build()
+        }.onErrorResume {
+            SaveNewWordResponse.newBuilder().apply {
+                failureBuilder.setErrorMassage(it.message)
+            }.build().toMono()
+        }
+
+    override fun saveNewWord(request: SaveNewWordRequest): Mono<SaveNewWordResponse> =
+        saveNewWord(request.toMono())
```

Comment:
I guess it's not mandatory to override this method since you did the same for similar method above

Reply:
yeap, my fault. I wanted to test it and forgot remove